src/lib/server/db/reports.channels.users.sql.ts

```diff
@@ -0,0 +1,23 @@
+import { pgTable, text, uuid } from 'drizzle-orm/pg-core';
+import { user } from './users.sql';
+import { channel } from './channels.sql';
+
+export const channelUserReport = pgTable('channel_user_reports', {
+    id: uuid('id').primaryKey().defaultRandom(),
+    postId: uuid('post_id')
+        .notNull()
+        .references(() => user.id),
+    channelId: uuid('channel_id')
+        .notNull()
+        .references(() => channel.id),
```

Comment:
This is technically redundant as the post already has a reference to its channel and they should always be the same

Reply:
It would be redundant here, but it shouldn't be. `channelId` stays for this one, since `postId` is supposed to be `userId`.